### PR TITLE
55, 56 therapy plan page , therapy goals page

### PR DESCRIPTION
**Pull request by fadeomar**
#### Issues this PR relates to
#55, #56
=======

_NOTE: REMEMBER NOT TO REVIEW UNLESS ALL ACCEPTANCE CRITERIA HAS BEEN TICKED OFF_

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist

Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)

- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/api-calls/therapists-clients.js [hunk 1]
```diff
@@ -25,4 +25,16 @@ const getTherapistByInviteToken = async ({ inviteToken, options }) => {
   }
 };
 
-export { getTherapistsClientsById, getTherapistByInviteToken };
+const getMyTherapy = async ({ options } = {}) => {
+  try {
+    const { data } = await axios.get(`${THERAPISTS_CLIENTS_BASE}/my-therapy`);
+    console.log({ data });
```
Comment (thejoefriel) [inline client/src/api-calls/therapists-clients.js:31]: remove console log

File: client/src/api-calls/therapists-clients.js [hunk 2]
```diff
@@ -25,4 +25,16 @@ const getTherapistByInviteToken = async ({ inviteToken, options }) => {
...
+    return { data };
+  } catch (error) {
+    console.log({ error });
```
Comment (thejoefriel) [inline client/src/api-calls/therapists-clients.js:34]: remove console log

File: client/src/pages/Client/MyTherapyGoals/index.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    };
+
+    getMyTherapy();
+  }, []);
+
+  return (
+    <>
+      <Title boldSection="Therapy Goals" lightSection="My" />
+      <Row mb="6" jc="space-between">
+        <Col w={[4, 12, 6]}>
```
Comment (thejoefriel) [inline client/src/pages/Client/MyTherapyGoals/index.js:29]: Can you make the desktop 8 instead please? I know yours is in line with wireframes but I feel 8 will actually look better now seeing it on desktop

File: server/modules/therapist-client/model/find.js [hunk 4]
```diff
@@ -34,4 +34,25 @@ const findTherapistByInviteToken = async (inviteToken, client) => {
...
+      therapy_background,
+      therapy_goals,
+      therapist_bio,
+      therapist_intro,
+      therapist_message
+    FROM therapist_clients
+    WHERE client_user_id = $1
+  `;
+
+  const res = await query(sql, values, client);
```
Comment (thejoefriel) [inline server/modules/therapist-client/model/find.js:50]: I don't think you need client here in this file as you're not doing a series of queries and also you don't feed it through as an argument from the use case

**Review by thejoefriel — CHANGES_REQUESTED** [review]

File: client/src/api-calls/therapists-clients.js (hunk 1 above)
Comment (fadeomar) [reply client/src/api-calls/therapists-clients.js:31]: done

File: client/src/api-calls/therapists-clients.js (hunk 2 above)
Comment (fadeomar) [reply client/src/api-calls/therapists-clients.js:34]: done

File: client/src/pages/Client/MyTherapyGoals/index.js (hunk 3 above)
Comment (fadeomar) [reply client/src/pages/Client/MyTherapyGoals/index.js:29]: done

File: server/modules/therapist-client/model/find.js (hunk 4 above)
Comment (fadeomar) [reply server/modules/therapist-client/model/find.js:50]: done

**Review by thejoefriel — APPROVED** [review]

**Pull request merged**